torch/testing/_deprecated.py

```diff
@@ -84,3 +86,21 @@ def assert_allclose(
         check_is_coalesced=False,
         msg=msg or None,
     )
+
+
+for name in _dtype_getters.__all__:
+    if name.startswith("_"):
+        continue
+
+    fn = getattr(_dtype_getters, name)
+
+    if name == "get_all_math_dtypes":
+        instructions = (
+            f"For CUDA devices, the call can be replaced with {fn('cuda')}. "
+            f"For all other devices, it can be replaced with {fn('cpu')}."
+        )
+    else:
+        instructions = f"The unparametrized call can be replaced with {fn()}"
```

Comment:
Let's include the names of the functions:

"The call to FOO can be replaced by enumerating the dtypes DTYPES"